README.md

```diff
@@ -0,0 +1,43 @@
+# ticket-booking
+
+
+### Additional assumptions
+- Requested time interval for point 1. of the scenario can be at max 1 week.
+- Search for movie screenings for point 1. of the scenario can apply only to future screenings.
+- Each row has the same number of seats
+- Reservation expiration date is set to 5 minutes after booking the seats
+- All tickets for single reservation must have the same currency
+
+---
+
+### How to build and run
+
+To build and run the app you can use provided bash script
+```
+./buildAndRun.sh
+```
+Or you can use mvnw directly
+```
+./mvnw spring-boot:run -Dspring-boot.run.profiles=dev
+```
+
+System uses in-memory database with initialized dummy data.
+
+---
+You can use useCase.sh bash script to execute required use case.
+It performs 3 REST calls and prints responses to the console.
+```
+./useCase.sh
+```
+
+Example output:
+```agsl
+1. Fetch movie screenings that start between 15:20 and 18:00 on 05.04.2023: 
+[{"movieTitle":"Django","screeningTimes":[{"screeningId":"4941fe3f-611b-48a2-b31d-ed2fc551069f","start":"2023-04-05T16:30:00","end":"2023-04-05T18:00:00"}]},{"movieTitle":"Hateful 8","screeningTimes":[{"screeningId":"cd174523-4630-4f36-9611-5a78d663e15a","start":"2023-04-05T15:30:00","end":"2023-04-05T17:00:00"},{"screeningId":"fd612a91-db41-4bc3-a794-f8c1a3d8e1d2","start":"2023-04-05T17:30:00","end":"2023-04-05T19:00:00"}]},{"movieTitle":"Pulp Fiction","screeningTimes":[{"screeningId":"a80be8de-0c61-4e0a-a3e5-95b055845c18","start":"2023-04-05T16:00:00","end":"2023-04-05T17:30:00"}]}]
```

Comment:
Fajnie byłoby to przepuścić przez jakiś formater (jq?).

Reply:
Rzeczywiście byłoby czytelniej, sorka :)